README.md

```diff
@@ -257,3 +257,7 @@ Have something you want to discuss, but you're not sure it warrants an issue? Fe
 
 - Thanks to [Chris Kempson](https://github.com/chriskempson) for the original concept and implementation.
 - Color wheel icon thanks to [Color icons created by Nikita Golubev - Flaticon](https://www.flaticon.com/free-icons/color).
+
+### Former admins
+
+- [@joshgoebel](https://github.com/joshgoebel)
```

Comment:
@joshgoebel are you ok with having your details here?

Reply:
I've removed this from the README.md since I haven't gotten permission to add it, but let us know anytime and we will add it.